pl_examples/bug_report/bug_report_model.ipynb

```diff
@@ -0,0 +1,268 @@
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "w4_TYnt_keJi"
+      },
+      "source": [
+        "import os\n",
+        "\n",
+        "import torch\n",
+        "from torch.utils.data import DataLoader, Dataset\n",
+        "\n",
+        "from pytorch_lightning import LightningModule, Trainer"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "XrJDukwPtUnS"
+      },
+      "source": [
+        "---\n",
+        "## Data\n",
+        "Random data is best for debugging. If you needs special tensor shapes or batch compositions or dataloaders, modify as needed"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "hvgTiaZpkvwS"
+      },
+      "source": [
+        "class RandomDataset(Dataset):\n",
+        "    def __init__(self, size, num_samples):\n",
+        "        self.len = num_samples\n",
+        "        self.data = torch.randn(num_samples, size)\n",
+        "\n",
+        "    def __getitem__(self, index):\n",
+        "        return self.data[index]\n",
+        "\n",
+        "    def __len__(self):\n",
+        "        return self.len"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "sxVlWjGhl02D"
+      },
+      "source": [
+        "num_samples = 10000"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "V7ELesz1kVQo"
+      },
+      "source": [
+        "class BoringModel(LightningModule):\n",
+        "    def __init__(self):\n",
+        "        super().__init__()\n",
+        "        self.layer = torch.nn.Linear(32, 2)\n",
+        "\n",
+        "    def forward(self, x):\n",
+        "        return self.layer(x)\n",
+        "\n",
+        "    def training_step(self, batch, batch_idx):\n",
+        "        loss = self(batch).sum()\n",
+        "        self.log(\"train_loss\", loss)\n",
+        "        return {\"loss\": loss}\n",
+        "\n",
+        "    def validation_step(self, batch, batch_idx):\n",
+        "        loss = self(batch).sum()\n",
+        "        self.log(\"valid_loss\", loss)\n",
+        "\n",
+        "    def test_step(self, batch, batch_idx):\n",
+        "        loss = self(batch).sum()\n",
+        "        self.log(\"test_loss\", loss)\n",
+        "\n",
+        "    def configure_optimizers(self):\n",
+        "        return torch.optim.SGD(self.layer.parameters(), lr=0.1)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ubvW3LGSupmt"
+      },
+      "source": [
+        "---\n",
+        "## Define the test\n",
+        "NOTE: in colab, set progress_bar_refresh_rate high or the screen will freeze because of the rapid tqdm update speed."
```

Comment:
```suggestion
        "NOTE: in colab, set progress_bar_refresh_rate high or the screen will freeze because of the rapid tqdm update speed."
```
It is deprecated and being used. Remove this instead

Reply:
done!